awesomeConfig.cmake

```diff
@@ -4,7 +4,7 @@ set(PROJECT_AWE_NAME awesome)
 # `git describe` later.
 set(VERSION devel)
 
-set(CODENAME "The Fox")
+set(CODENAME "Harder, Better, Faster, Stronger")
```

Comment:
I remember you asking for a RC. Is this codename for the RC or for the final release? (I assume for the final release)